AppKit/CoreAnimation/CPAnimationContext.j

```diff
@@ -162,14 +162,19 @@ CPLog.debug(_cmd + "context stack =" + _CPAnimationContextStack);
     var completionFunction = function()
     {
         if (needsFrameTimer)
+        {
             [self stopFrameUpdaterWithIdentifier:objectId];
+            [anObject _setForceDOMUpdates:YES];
```

Comment:
Sorry but I don't understand where else you want me to force updates. This should only be set when the animation used "live" setFrameXXX with _inhibitDOMUpdates set as frame rect is, at the end of the animation, set to the final value but without having impacted the DOM (so, simply calling setFrameXXX has no effect as each method begins with ```if (CGRectEqualToRect(_frame, aFrame))``` (or similar for other methods). Maybe I'm missing something somewhere...  ;-)  Do you have any clue for me ?